lib/ansible/module_utils/facts/hardware/__init__.py

```diff
@@ -0,0 +1,32 @@
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.module_utils.facts.hardware import aix
+from ansible.module_utils.facts.hardware import darwin
+from ansible.module_utils.facts.hardware import dragonfly
+from ansible.module_utils.facts.hardware import freebsd
+from ansible.module_utils.facts.hardware import hpux
+from ansible.module_utils.facts.hardware import hurd
+from ansible.module_utils.facts.hardware import linux
+from ansible.module_utils.facts.hardware import netbsd
+from ansible.module_utils.facts.hardware import openbsd
+from ansible.module_utils.facts.hardware import sunos
```

Comment:
Unless I'm forgetting some wrinkle in get_all_subclasses, we should be able to move verything to something like hardware.base, though.  That way we keep the code out of the __init__.py.